packages/oas-utils/src/helpers/redirect-to-proxy.ts

```diff
@@ -35,22 +47,28 @@ export const isRelativePath = (url: string) => {
   return true
 }
 
-/** Returns false for requests to localhost, relative URLs, if no proxy is defined … */
+/**
+ * Returns false for requests to localhost, relative URLs, if no proxy is defined …
+ **/
 export function shouldUseProxy(proxy?: string, url?: string): boolean {
-  // No proxy or url
-  if (!proxy || !url) {
-    return false
-  }
+  try {
+    // No proxy or url
+    if (!proxy || !url) {
+      return false
+    }
 
-  // Relative URLs
-  if (isRelativePath(url)) {
-    return false
-  }
+    // Relative URLs
+    if (isRelativePath(url)) {
+      return false
+    }
+
```

Comment:
```suggestion
    }

  // Requests to a local proxy
  if (isLocalProxy(proxy) {
    return true
  }

```

Reply:
Yea if its deployed then they should not pass in a localhost proxy. Its only for local dev

Reply:
why can't we just allow the user to pass it in? If they pass in a localhost on a domain it won't work